### ISSUE TITLE
Something that could be added to 40 lines and maybe other solo modes aswell

### ISSUE THREAD

**Issue by 1dullz** [tetrio/issues#1080]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

No it is a suggestion!

### Describe the solution you'd like

Basically you can make it where after you finish 40 lines you make it where it tells you how many lines you made it after you loose (if you loose) so you can know if you are getting better or not and I am saying this as a beginner at Tetris that is my suggestion, thank you!

### Describe alternatives you've considered

A high score system aswell

### Additional context

Nothing more I need to say.
